webapp/client/App.js

```diff
@@ -1,36 +1,33 @@
 import React from 'react';
 import PopularMovies from './PopularMovies';
 import Search from './Search'
+  ;
+
 class App extends React.Component {
-  constructor(props){
+  constructor(props) {
     super(props);
     this.state = {
       popular: {},
-      searchState: false
-    }
-    }
-    componentWillMount(){
-      fetch('/popular')
-      .then(results => {
-        return results.json()
-      }).then(data=> {
-        this.setState({popular: data})
-        //console.log(this.state.popular.results[0].title)
-      })
-    }
+      searchState: false,
+    };
+  }
+  componentWillMount() {
+    fetch('/popular')
+      .then(results => results.json()).then((data) => {
```

Comment:
if you send your results back as json (which you should), you wont need to call results.json()